ReservacionDeVuelos/App.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Providers from './navigation';
+
+const App = () => {
+  return <Providers/>;
+}
+
+export default App;
```

Comment:
Add new blank screen

Reply:
thank you for your observation, we already added the blank line at the end of the file and removed all the comments :)